swap_meet/vendor.py

```diff
@@ -1,2 +1,87 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        
+        self.inventory.remove(item)
+        return item
+    
+    def get_by_id(self, id):
+        try:
+            for i in range(len(self.inventory)):
+                if self.inventory[i].id == id:
+                    return self.inventory[i]
+        except:
+            return None
```

Comment:
Surprisingly, this implicitly returns `None` in the case where it doesn't find a particular `id`! You can remove the `try`/`except`, since it shouldn't be able to raise an exception, and `return None` if the loop ends without finding an item.